src/pages/Address.vue

```diff
@@ -39,15 +39,22 @@ export default {
         this.contract = await this.$contractManager.getContract(this.address)
         this.isVerified = this.contract.verified;
       }
-      let strBalance = web3.utils.fromWei(account.balance);
-      strBalance = `${strBalance.substring(
-        0,
-        strBalance.indexOf(".") + 5
-      )} TLOS`;
-      this.balance = strBalance;
+
+      this.balance = this.getBalanceDisplay(account.balance);
       this.telosAccount = account.account;
       this.isContract = account.code.length > 0;
     },
+    getBalanceDisplay(balance) {
+      let strBalance = web3.utils.fromWei(balance);
+      const decimalIndex = strBalance.indexOf(".");
+      if (decimalIndex > 0) {
+        strBalance = `${strBalance.substring(
+          0,
+          strBalance.indexOf(".") + 5
+        )}`;
+      }
+      return `${strBalance} TLOS`;
+    },
```

Comment:
I'm wondering if you can just use something like the following (replacing precision/max/min as needed):
```
     return  Number(amount).toLocaleString('en-US', {
        style: 'decimal',
        maximumFractionDigits: token.precision,
        minimumFractionDigits: token.precision
      });
      ```